modules/tiles/src/tileset/tileset-3d.js

```diff
@@ -551,7 +555,7 @@ export default class Tileset3D {
   }
 
   _unloadTile(tile) {
-    this.gpuMemoryUsageInBytes -= tile.content.byteLength || 0;
+    this.gpuMemoryUsageInBytes -= (tile.content && tile.content.byteLength) || 0;
```

Comment:
It would be good to maintain a `gpuMemoryUsageInBytes` on the tile so we didn't need to rake into its internals here. I am not sure `content` would usually be a typed array.

Reply:
I am not sure what you mean? All I added here was a check that `tile.content` exists (I experienced a race condition in which the content was already gone) .

Reply:
Sorry I see that I wasn't reviewing your change, I was reviewing the original code.

Reply:
Ah ok. we can the maybe handle this later on.